<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Identity Service Resilience Test Suite (RES-001 to RES-005)" tests="1" skipped="0" failures="1" errors="0" timestamp="2025-09-19T16:32:30" hostname="mac14.local" time="0.001">
  <properties/>
  <testcase name="initializationError" classname="com.focushive.identity.resilience.ResilienceTestSuite" time="0.001">
    <failure message="org.junit.platform.suite.engine.NoTestsDiscoveredException: Suite [com.focushive.identity.resilience.ResilienceTestSuite] did not discover any tests" type="org.junit.platform.suite.engine.NoTestsDiscoveredException">org.junit.platform.suite.engine.NoTestsDiscoveredException: Suite [com.focushive.identity.resilience.ResilienceTestSuite] did not discover any tests
	at java.base@21.0.8/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:184)
	at java.base@21.0.8/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base@21.0.8/java.util.Iterator.forEachRemaining(Iterator.java:133)
	at java.base@21.0.8/java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1939)
	at java.base@21.0.8/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
	at java.base@21.0.8/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base@21.0.8/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:151)
	at java.base@21.0.8/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:174)
	at java.base@21.0.8/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base@21.0.8/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596)
</failure>
  </testcase>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
